README.md

```diff
@@ -59,6 +59,62 @@ Access web locally:
 Check `Makefile` for shell commands if you want to run them with modified
 parameters.
 
+### .zoo.yml file support for repositories
+
+.zoo.yml is a file that can be placed in a repository which zoo will if found  while scanning repositories
```

Comment:
This sentence is hard to understand and there is a missing word. I would probably change it to something like

> Repositories scanned by the Zoo may contain the `.zoo.yml` file. This file contains additional information about the scanned repository and how and where it's used in production. If the Zoo finds this file in the root of the project, it will read it and create a `Service` or a `Library` within the Zoo with the provided information based on the collected data automatically. Otherwise the data have to be added to the Zoo manually.
>
> Here is an example of this file:

And I would put just an example here, not the schema itself which you would have to keep synced with the one you have in the source code. The example doesn't have to include all fields, you can add a link to full example/schema from here. The important thing is that developers won't understand the fields, I myself don't know what `sentry_project` should be... should it be full link or just the part after base URL? And what about `slack_channel`, should it be with `#` or without? You get the point. This has to be documented.

Reply:
Yes you are right here. I'm really bad at writing documentation :)

Reply:
Your explanation was perfect so i just copied it if you dont mind :)